install-prereqs.sh

```diff
@@ -61,9 +62,8 @@ else
     pre-commit install
     pre-commit install --hook-type commit-msg
 
-    ln -s `pwd`/sdcm $(python -c "import site; print site.getsitepackages()[0]")/sdcm
+    ln -s $PWD/sdcm $(python -c "import site; print(site.getsitepackages()[0])")/sdcm
```

Comment:
updated to use python3 syntax

Reply:
I'm quite sure we don't need this symbolic link any more, please remove this one